Composer/packages/client/src/components/ProjectTree/index.tsx

```diff
@@ -146,11 +136,7 @@ export const ProjectTree: React.FC<IProjectTreeProps> = props => {
   }
 
   const onRenderShowAll = () => {
-    return (
-      <ActionButton css={addButton(1)} iconProps={addIconProps} onClick={openNewTriggerModal}>
-        <span ref={addNewTriggerRef}>{formatMessage('New Trigger ..')}</span>
-      </ActionButton>
-    );
+    return null;
   };
```

Comment:
Can this just be removed now?

Reply:
It doesn't look like it. A default element "Show All" shows up.